vbfml/scripts/err_analysis

```diff
@@ -378,5 +384,67 @@ def evaluate(ctx, input_dir: str, sequence_type: str) -> None:
     model.evaluate(sequence)
 
 
+@cli.command()
+@click.pass_context
+@click.option(
+    "-i", "--input-dir", required=True, multiple=True, help="files to compare"
+)
+@click.option(
+    "-n",
+    "--number-epochs",
+    required=False,
+    type=int,
+    default=5,
+    help="The number of last epochs considered in the statistics",
+)
+def compare_hist(ctx, input_dir: str, number_epochs: int, save_name: str) -> None:
```

Comment:
Silly question: Where does `save_name` come from, it is not a command line argument right?

Reply:
very good silly question, that was a mistake from when I switch branch
I corrected it by adding the click option --save-name